src/lib/utils.ts

```diff
@@ -14,3 +14,12 @@ const endsWith = (str: string, suffix: string) => str.indexOf(suffix, str.length
  */
 export const normalizeKeyPrefix = (keyPrefix: string) =>
   keyPrefix.length > 0 && !endsWith(keyPrefix, '.') ? `${keyPrefix}.` : keyPrefix;
+
+/**
+ * Logs that there was an error in retrieving the translation map.
+ */
+export const logTranslationsNOK = () => {
+  if (process.env.NODE_ENV !== 'production') {
+    console.log('[react-message-source] Failed to get proper translations. Setting an empty translation map.');
+  }
+};
```

Comment:
I'm not sure if we like to write a log for this. Maybe it would be better if we mention in the documentation that RMS will failback to `no-translations` in case of an error.